python/cudf/cudf/_lib/cpp/io/parquet.pxd

```diff
@@ -191,6 +199,8 @@ cdef extern from "cudf/io/parquet.hpp" namespace "cudf::io" nogil:
         void set_row_group_size_rows(size_type val) except +
         void set_max_page_size_bytes(size_t val) except +
         void set_max_page_size_rows(size_type val) except +
+        void enable_write_v2_headers(bool val) except +
```

Comment:
I realize it isn't the fault of this current PR, but one does wish `enable_write_v2_headers` were named `set_write_v2_headers`.

Reply:
We use enable_ for bool options, so this should be consistent (for better or for worse, apparently).